Vagrant.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
```

Comment:
Sorry. Don't know Chef. :(

I've been using bash script to setup development-only boxes for single machine systems. So this patch is simply an adaptation of what I use.

However I don't intend to change your stablished workflow. I just had the problem with chef, bypassed it and I'm sharing what worked for me.

If make sense to you, I could update the patch to keep both provisioners while chef is not fixed. They would be accessible via the `--provision-with` parameter.